app/styles/live-sidebar.module.css

```diff
@@ -0,0 +1,195 @@
+.sidebar {
+  padding: 1rem 0;
+  width: 100%;
+  background-color: var(--color-sidebar);
+  height: 100%;
+  z-index: 2;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.sidebar__accordian-button {
+  all: unset;
+  display: none;
+}
+
+.sidebar__body {
+  width: 90%;
+  height: 327px;
+  overflow: auto;
+}
+
+.sidebar__body::-webkit-scrollbar {
+  width: 5px;
+}
+
+.sidebar__body::-webkit-scrollbar-thumb {
+  background: var(--color-darkgrey);
+  border-radius: 10px;
+}
+
+.sidebar__participants {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: white;
+  text-align: center;
+  padding: 0.5rem 2rem;
+  margin-bottom: 1rem;
+  border-radius: 2rem;
+  color: var(--color-navyblue);
+  font-style: normal;
+  font-weight: 800;
+  font-size: 22px;
+  line-height: 26px;
+  box-sizing: border-box;
+}
+
+.sidebar__participants--mobile {
+  display: none;
+  width: 100%;
+  background: rgba(255, 255, 255, 0);
```

Comment:
Can we please use color variable here and make use of hex values to maintain the consistency?